notebooks/Emukit-tutorial-bayesian-optimization-benchmark.ipynb

```diff
@@ -82,231 +82,31 @@
-      "Optimization restart 1/1, f = 98.91095438834384\n",
-      "Optimization restart 1/1, f = 20.785966078259044\n",
-      "Optimization restart 1/1, f = 37.680169736604405\n",
-      "Optimization restart 1/1, f = 43.42073131574933\n",
-      "Optimization restart 1/1, f = 49.259617085525356\n",
-      "Optimization restart 1/1, f = 55.97390870794924\n",
-      "Optimization restart 1/1, f = 61.67806077414744\n",
-      "Optimization restart 1/1, f = 67.3342863518653\n",
-      "Optimization restart 1/1, f = 73.02141978867155\n",
-      "Optimization restart 1/1, f = 78.66087865527318\n"
+      " /Users/marpulli/git/emukit/emukit/examples/models/random_forest.py:42: DataConversionWarning:A column-vector y was passed when a 1d array was expected. Please change the shape of y to (n_samples,), for example using ravel().\n"
      ]
     },
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Optimization restart 1/1, f = 84.60608464011534\n",
-      "Optimization restart 1/1, f = 90.13194449420432\n",
-      "Optimization restart 1/1, f = 96.14908161701135\n",
-      "Optimization restart 1/1, f = 101.71274938730758\n",
-      "Optimization restart 1/1, f = 107.96094925477432\n",
-      "Optimization restart 1/1, f = 113.61853029459846\n",
-      "Optimization restart 1/1, f = 119.76750654925954\n",
-      "Optimization restart 1/1, f = 125.25298134788359\n",
-      "Optimization restart 1/1, f = 130.7057592698655\n",
-      "Optimization restart 1/1, f = 136.36344378889436\n",
-      "Optimization restart 1/1, f = 143.6180229019242\n",
-      "Optimization restart 1/1, f = 149.1026130923552\n",
-      "Optimization restart 1/1, f = 1927569.4351349822\n",
-      "Optimization restart 1/1, f = 5146467.01905448\n",
-      "Optimization restart 1/1, f = 6550320.696311793\n",
-      "Optimization restart 1/1, f = 7182412.44557667\n",
-      "Optimization restart 1/1, f = 7665949.962034388\n",
-      "Optimization restart 1/1, f = 7819318.668270855\n",
-      "Optimization restart 1/1, f = 8162081.264127023\n",
-      "Optimization restart 1/1, f = 10072031.740549482\n",
-      "Optimization restart 1/1, f = 10072206.72559119\n",
-      "Optimization restart 1/1, f = 10123486.59846111\n",
-      "Optimization restart 1/1, f = 11358716.217228113\n",
-      "Optimization restart 1/1, f = 12614933.888620663\n",
-      "Optimization restart 1/1, f = 12824941.717794457\n",
-      "Optimization restart 1/1, f = 5031473.484086115\n",
-      "Optimization restart 1/1, f = 92.48775577692987\n",
-      "Optimization restart 1/1, f = 97.55579913952919\n",
-      "Optimization restart 1/1, f = 103.79995195601342\n",
-      "Optimization restart 1/1, f = 109.60497253687181\n",
-      "Optimization restart 1/1, f = 114.19563285157699\n",
-      "Optimization restart 1/1, f = 118.25661469344114\n",
-      "Optimization restart 1/1, f = 121.95962284675001\n"
+     "ename": "AttributeError",
```

Comment:
The notebooks seems to throw an error.

Reply:
Oops, I seem to have not rerun the notebook with the latest code, I'll rerun it.